scipy/optimize/__init__.py

```diff
@@ -247,8 +247,11 @@
    optimize.linprog-simplex
```

Comment:
the permissions change on this file looks incorrect

Reply:
Fixed.